casadm/cas_lib.c

```diff
@@ -1133,8 +1133,12 @@ int get_cache_mode(int ctrl_fd, unsigned int cache_id, int *mode)
 	cmd_info.cache_id = cache_id;
 
 	if (ioctl(ctrl_fd, KCAS_IOCTL_CACHE_INFO, &cmd_info) < 0)
-		return FAILURE;
+	{
+		if (cmd_info.ext_err_code == OCF_ERR_CACHE_PASSIVE)
+			cas_printf(LOG_ERR, "Cannot get cache mode while cache %d is in passive mode.", cmd_info.cache_id);
```

Comment:
I'm not sure if this is the time already (as it may change again soon ;) ) but we probably need to rename all "passive" to "standby".